app/assets/scss/application.scss

```diff
@@ -84,3 +84,7 @@ $path: "/suppliers/static/images/";
   @include core-19;
   margin: (1.5 * $gutter) 0 (2 * $gutter);
 }
+
+.padding-bottom-small {
```

Comment:
I think I introduced this to the buyer frontend a little while ago (it was a thing in the prototype that I copied over but probably shouldn't have), and I've been trying to backtrack on it and removed it from a couple of places recently.  But it seems to be a useful thing (Caroline added a bunch more in buyer frontend recently), so let's stick with it for now.